contents/teams/marketing/objectives.mdx

```diff
@@ -1,73 +1,24 @@
-- **Contributors:** Andy, Ian, Lior
-- **What we'll ship:** [See issue](https://github.com/PostHog/posthog.com/issues/7517#issuecomment-2015715230)
-- **We'll know we're successful when:** We've shipped priority guides.
-
-### Scale newsletter (James)
-
-- **Rationale:** We've successfully reached 10k subs and people like the newsletter. We think it can be a big driver of word of mouth growth if we can scale it further.
-- **Contributors:** James Temperton, Andy, Ian, Lior, James Hawkins. 
-- **What we'll ship: (WIP)** [See issue](https://github.com/PostHog/meta/issues/192)
-- Transition to 1 new issue per week
-- Improve content promotion on Twitter and LinkedIn
-- Trade way more recommendations with other newsletters
-- Get at least 1 new (or old) post to go viral
-- Improve visibility on the website
-- Do more issues that use our network
-- Land a big interview
-- Continue newsletter sponsorships
-- **We'll know we're successful when:** We can consistently ship great newsletters every week while maintaining a strong open rate (45% or more) and subscriber growth is accelerating.
-
-### Level up branding (Lottie)
-
-- **Rationale:** People love our brand, but it will go stale if don't refresh and create new things that spark joy and conversation.
-- **Contributors:** Lottie working with Website & Docs
-- **What we'll ship:** [See Lottie's issue](https://github.com/PostHog/meta/issues/191)
-- **We'll know we're successful when:** People continue to talk about our brand and sharing the awesome new things we ship.
-
-### Hiring plan for marketing team (Charles)
-
-- **Rationale:** Demand on team is rapidly expanding as we launch new products, but this needs to be tempered with hiring correct roles + not over hiring.
-- **Contributors:** Charles
-- **What we'll ship:** 12-month hiring plan + budget
-- **We'll know we're successful when:** We've hired cracked new marketer(s).
+- **Rationale:** We're well-known for SaaS, but less so for mobile. Open beta of mobile replay is an opportunity to grow. 
+- **Contributors:** All (Owned by Lior)
+- **We'll know we're successful when:** When we see accelerating usage of mobile SDKs and word-of-mouth mentions
 
 ## Side quests
 
-- [Figure out if we should invest further in framework tutorial videos](https://github.com/PostHog/posthog.com/issues/7908) (Lior)
-- Ship awesome technical blogs, such as rewriting our pipeline in Rust, Mr Blobby, HogQL insight rewrites. (Ian)
-
-## Metrics we care about
-
-- New org signups
-- Newsletter signups
-- Organic SEO visits
-
-<PrivateLink url="https://us.posthog.com/project/2/dashboard/51661">Marketing dashboard</PrivateLink>
+- Landing pages for use cases (Andy)
+- More data warehouse tutorials and templates (Ian)
+- Refreshing our paid ads strategy (Andy & Charles)
+- Nailing socials and content distribution (James)
+- Migration guides for popular alternatives (Ian)
+- Special edition T-shirts and merch collabs (Lottie)
```

Comment:
@charlescook-ph @lottiecoxon feel free to remove for something else. This was discussed as an idea, but not super attached to this if you'd rather work on something else.

Reply:
Reviews crossed paths - I've suggested an edit (which we can iterate on when Lottie is back if needed!)